components/led.py

```diff
@@ -72,10 +72,18 @@ def want_note(self) -> None:
         self.set_colour(HsvColour.MAGENTA.value)
         self.set_state(PatternState.FLASH)
 
-    def holding_note(self) -> None:
+    # def holding_note(self) -> None:
```

Comment:
will the holding_note led state be used/necessary?

Reply:
No, I don't think so. It'll be switching to in_range and out_of_range states once holding the note